scripts/bountiful.ash

```diff
@@ -339,6 +339,121 @@ string addBountyToQueue(monster opp, boolean speculate) {
   return "";
 }
 
+/**
+* Determines if desired monster is in our received fax.
+* If it is not, send fax to clear inventory
+* @param {monster} enemy - desired fax target
+* @returns {boolean} - true if have photocopy of desired monster
+*/
+boolean checkFax(monster enemy)
+{
+	if(item_amount($item[photocopied monster]) == 0)
+	{
+		cli_execute("fax receive");
+	}
+
+	if(get_property("photocopyMonster") == enemy.to_string())
+	{
+		return true;
+	}
+
+	cli_execute("fax send");
+	return false;
+}
+
+/**
+* Attempts to fax and fight a monster. 
+* Will try CheeseFax first and if fails attempts EasyFax
+* @param {monster} enemy - desired fax target
+* @returns {boolean} - true if desired monster was faxed and fought
+*/
+boolean handleFaxMonster(monster enemy)
+{
+	if(get_property("_photocopyUsed").to_boolean())
+	{
+		return false;
+	}
+  if(contains_text(get_property("_bountiful.FailedFaxes"), enemy))
+  {
+    return false;
+  }
+	if(!is_unrestricted($item[deluxe fax machine]))
+	{
+		return false;
+	}
+	if(item_amount($item[Clan VIP Lounge Key]) == 0)
+	{
+    print("Faxing is enabled in Bountiful but you don't have a Clan VIP Lounge Key!", "red");
```

Comment:
i know we don't have a linter but it slightly annoys me to have this print (and most subsequent prints) being not indented the same way as, say, line 382. can we make those indents equivalent to ensure it is easy to parse which node of the tree stuff is in?

Reply:
Good call. I wish ash had a linter